core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -8922,6 +8922,183 @@ LogicalProject(EMPNO=[$0], ENAME=[$1], JOB=[$2], MGR=[$3], HIREDATE=[$4], SAL=[$
       LogicalAggregate(group=[{0}])
         LogicalProject(NAME=[$1])
           LogicalTableScan(table=[[CATALOG, SALES, DEPT]])
+]]>
+        </Resource>
+    </TestCase>
+    <TestCase name="testSortCopyTranspose1">
```

Comment:
I think this xml block (`testSortCopyTranspose1`) does not belong to any actual unit test

Reply:
It belonged to an old test indeed, I just removed it.